blocks/log.py

```diff
@@ -203,6 +208,16 @@ def _check_time(self, time):
         if not isinstance(time, int) or time < 0:
             raise ValueError("time must be a positive integer")
 
+    def to_dataframe(log):
```

Comment:
As Scrutinizer points out, this method and the next one probably need `self` as their first argument.